chore(pubsub): release 1.1.0

pradn opened the pull request:
Release a new version since there have been a few fixes.

plamut commented on pubsub/setup.py:
```diff
@@ -22,7 +22,7 @@
 name = "google-cloud-pubsub"
 description = "Google Cloud Pub/Sub API client library"
-version = "1.0.2"
+version = "1.0.3"
```
Just a thought - since we have a new feature (the `publisher.stop()` method), shouldn't that be a _minor_ version bump, i.e. `1.1.0`?

pradn replied:
Good point, will bump the minor.

plamut commented on pubsub/CHANGELOG.md:
```diff
@@ -4,6 +4,31 @@
+
+### Internal / Testing Changes
+- Refactor fake leaser test helper ([#9632](https://github.com/googleapis/google-cloud-python/pull/9632))
+- Add subscriber role test for streaming ([#9507](https://github.com/googleapis/google-cloud-python/pull/9507))
```
(nit)
AFAIK the style guide mandates that all release message end with a full stop?

plamut commented on pubsub/CHANGELOG.md:
```diff
@@ -4,6 +4,31 @@
+- Add stop method ([#9365](https://github.com/googleapis/google-cloud-python/pull/9365))
+
+### Dependencies
+- Add python 2 sunset banner to documentation ([#9036](https://github.com/googleapis/google-cloud-python/pull/9036))
```
(nit)
"python" should be capitalized.

plamut commented on pubsub/CHANGELOG.md:
```diff
@@ -4,6 +4,31 @@
+- Split large (mod)ACK requests into smaller ones ([#9594](https://github.com/googleapis/google-cloud-python/pull/9594))
+- Fix messages delivered multiple times despite a long ACK deadline ([#9525](https://github.com/googleapis/google-cloud-python/pull/9525))
+- Update batching and flow control parameters to be same as the other client libraries ([#9597](https://github.com/googleapis/google-cloud-python/pull/9597))
+- Add 'StreamingPullManager._should_terminate' ([#9335](https://github.com/googleapis/google-cloud-python/pull/9335))
```
(nit)
It's probably better to use backticks instead of single quotes, renders more nicely.

pradn commented:
Release is at https://github.com/googleapis/google-cloud-python/releases/tag/pubsub-1.0.3

plamut commented:
@pradn I manually changed the tag and release title to `1.1.0`, please check if a manual run is needed to publish the release to PyPI (the bot might be tempted to upload the 1.0.3 tag).

pradn commented:
Okay, I should have just made a new branch/PR with 1.0.1 to make it simple.

I deleted the 1.0.3 tag in the this repo. The 1.1.0 tag looks good. I added the "autorelease:tagged" flag back on to continue the release to PyPi.

Hopefully everything works now.

plamut commented:
It appears the bot will have to be run manually, the similar thing happened when releasing API Core recently.

(cc: @busunkim96 )

busunkim96 commented:
Released to pypi: https://pypi.org/project/google-cloud-pubsub/1.1.0
